test/e2e/data/e2e_conf.yaml

```diff
@@ -252,21 +252,21 @@ variables:
   # allowing the same e2e config file to be re-used in different Prow jobs e.g. each one with a K8s version permutation.
   # The following Kubernetes versions should be the latest versions with already published kindest/node images.
   # This avoids building node images in the default case which improves the test duration significantly.
-  KUBERNETES_VERSION_MANAGEMENT: "v1.23.3"
-  KUBERNETES_VERSION: "v1.23.3"
-  KUBERNETES_VERSION_UPGRADE_TO: "v1.23.3"
-  KUBERNETES_VERSION_UPGRADE_FROM: "v1.22.4"
+  KUBERNETES_VERSION_MANAGEMENT: "v1.24.0"
```

Comment:
I think we have to keep this version on v1.23.3 for now.

The problem is that only kind v0.13 is compatible with kindest/node:v1.24.0+ and we didn't bump the kind version yet.

(cc @fabriziopandini)

Reply:
ACK, done.

Reply:
With the latest v1.1.x release we should now be able to use v1.24 as mgmt cluster